.gitlab-ci.yml

```diff
@@ -1,12 +1,12 @@
 variables:
   CI_DOCKER_IMAGE: registry.ddbuild.io/images/docker:24.0.4-gbi-focal
-  DOCKER_CONTEXT: "."
+  DOCKER_CTX: "."
```

Comment:
`DOCKER_CONTEXT` is a reserved env var: https://docs.docker.com/engine/context/working-with-contexts/#use-a-different-context and was causing the following error:
```
ERROR: no builder "." found
```

Not sure why this was working before 🤔